modules/services/cloud-bench/main.tf

```diff
@@ -162,4 +162,10 @@ resource "aws_cloudformation_stack_set_instance" "stackset_instance" {
   deployment_targets {
     organizational_unit_ids = [for root in data.aws_organizations_organization.org[0].roots : root.id]
   }
+  operation_preferences {
+    failure_tolerance_count = 100
+    # failure_tolerance_percentage = 100
```

Comment:
Typo?

Reply:
Wanted to mention there is another option as percentage. should we remove those? @nkraemer-sysdig

Reply:
Oh I see. Yes, maybe lets remove it if we're not using percentages

Reply:
done. removed them.